R/mod_decision_automation.R

```diff
@@ -228,38 +229,83 @@ mod_decision_automation_server <- function(id, user){
       initial_selection <- NULL
     }
     initial_values <- purrr::map(initial_values, ~ .x %>% `[`(!. %in% c(0,1)) %>% round(2))
+    updated_values <- do.call(reactiveValues, initial_values)
+    
+    dec_divs <- reactive({
+      purrr::map(decision_lst, ~ div(
+        risk = risk_lbl(.x, input = FALSE),
+        class = if (!.x %in% auto_current()) "shinyjs-hide",
+        style = "width: 100%",
+        sliderInput(ns(risk_lbl(.x)), 
+                    .x, 0, 1, updated_values[[.x]],
+                    width = "100%", sep = .01)
+      ))
+    })
+    
+    col_divs <- reactive({
+      purrr::map2(decision_lst, color_updated(), ~ div(
+        style = "width: 33.3%",
```

Comment:
Back to 33.3%? That's cool, I just thought accommodating 4 categories (at 25%) still looked nice but 5 (at 20%) would probably start to converge labels above each color cat. Obviously, if this could scale from 2 cats (meaning a 50% width) to max of 4 cats (25%) would be ideal.

This is what 5 cats looks like at 33.3% for reference:

![image](https://github.com/pharmaR/riskassessment/assets/17878953/07e617ba-17fe-4f07-9ffd-de62ca823756)

Reply:
> Obviously, if this could scale from 2 cats (meaning a 50% width) to max of 4 cats (25%) would be ideal.

Let's do that then. I like it.